[2025-01-03T07:13:48.452+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: insert_data.call_clear_all_data scheduled__2025-01-02T00:00:00+00:00 [queued]>
[2025-01-03T07:13:48.461+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: insert_data.call_clear_all_data scheduled__2025-01-02T00:00:00+00:00 [queued]>
[2025-01-03T07:13:48.462+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2025-01-03T07:13:48.637+0000] {taskinstance.py:2191} INFO - Executing <Task(SQLExecuteQueryOperator): call_clear_all_data> on 2025-01-02 00:00:00+00:00
[2025-01-03T07:13:48.644+0000] {standard_task_runner.py:60} INFO - Started process 92 to run task
[2025-01-03T07:13:48.649+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'insert_data', 'call_clear_all_data', 'scheduled__2025-01-02T00:00:00+00:00', '--job-id', '184', '--raw', '--subdir', 'DAGS_FOLDER/insert_data.py', '--cfg-path', '/tmp/tmpckzfn5ym']
[2025-01-03T07:13:48.653+0000] {standard_task_runner.py:88} INFO - Job 184: Subtask call_clear_all_data
[2025-01-03T07:13:48.715+0000] {task_command.py:423} INFO - Running <TaskInstance: insert_data.call_clear_all_data scheduled__2025-01-02T00:00:00+00:00 [running]> on host 403e7aadd7ee
[2025-01-03T07:13:48.832+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='agushchin' AIRFLOW_CTX_DAG_ID='insert_data' AIRFLOW_CTX_TASK_ID='call_clear_all_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-02T00:00:00+00:00'
[2025-01-03T07:13:48.834+0000] {sql.py:276} INFO - Executing: CALL stage.clear_all_data()
[2025-01-03T07:13:48.842+0000] {base.py:83} INFO - Using connection ID 'postgres-db' for task execution.
[2025-01-03T07:13:48.855+0000] {base.py:83} INFO - Using connection ID 'postgres-db' for task execution.
[2025-01-03T07:13:48.872+0000] {sql.py:450} INFO - Running statement: CALL stage.clear_all_data(), parameters: None
[2025-01-03T07:13:48.906+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=insert_data, task_id=call_clear_all_data, execution_date=20250102T000000, start_date=20250103T071348, end_date=20250103T071348
[2025-01-03T07:13:48.942+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-01-03T07:13:48.974+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
